base/random.jl

```diff
@@ -17,7 +17,7 @@ export srand,
        randperm, randperm!,
        randcycle, randcycle!,
        AbstractRNG, MersenneTwister, RandomDevice,
-       GLOBAL_RNG, randjump
+       GLOBAL_RNG, globalRNG, randjump
```

Comment:
Should probably call it `globalrng`, otherwise the capital RNG makes it look like camel case.